src/libguestfish.sh

```diff
@@ -35,14 +36,17 @@ coreos_gf() {
 coreos_gf_run_mount() {
     coreos_gf_launch "$@"
     coreos_gf run
-    local root=$(coreos_gf findfs-label root)
+    local root
```

Comment:
Just curious, is there an advantage of declaring and defining local variable in separate line? If yes, maybe we should also change it at places like https://github.com/coreos/coreos-assembler/pull/193/files#diff-496abb20f29d0275d6697b14356e60e1R131

Reply:
Yeah, it's one of the things that `shellcheck` flags

https://github.com/koalaman/shellcheck/wiki/SC2155

Admittedly, I did notice that it didn't always flag what appeared to be "offending" code, but didn't dig into the why.

Reply:
I could be more thorough and find all the `local` assignments and change them to match, if that seems reasonable.

Reply:
I see, it's only a problem if one is executing a subprocess basically, in other words `local vmpreparedir=${workdir}/tmp/supermin.prepare` is fine.

Reply:
Ah, yup...that looks like the reasoning!